onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -906,6 +908,30 @@ def remove_unused_constant(self):
         if len(unused_nodes) > 0:
             logger.debug(f"Removed unused constant nodes: {len(unused_nodes)}")
 
+    def get_subgraph_inputs_of_node(self, node):
+        """
+        Get inputs to all nodes in all subgraphs of a node
+        """
+        subgraph_nodes_inputs = set()
+        for attr in node.attribute:
+            if attr.type == AttributeProto.GRAPH:
+                child_nodes = attr.g.node
+                for child_node in child_nodes:
```

Comment:
nit: Add a comment that we do handle subgraph of child nodes. This function only handle one level subgraphs.